clang/include/clang/Driver/Options.td

```diff
@@ -3539,6 +3539,10 @@ def fno_builtin : Flag<["-"], "fno-builtin">, Group<f_Group>,
 def fno_builtin_ : Joined<["-"], "fno-builtin-">, Group<f_Group>,
   Visibility<[ClangOption, CC1Option, CLOption, DXCOption]>,
   HelpText<"Disable implicit builtin knowledge of a specific function">;
+def ffeature_availability_EQ : Joined<["-"], "ffeature-availability=">, Group<f_Group>,
```

Comment:
Silly question: why do we need this flag if all the features are defined in a header?

Reply:
I added this flag for testing purposes before we decided on the availability domain definition spec. I can remove it as it's not needed anymore. I left it in as I thought it could be handy when testing or debugging the feature.